### ISSUE TITLE
Large <input type="date"> in Edge/Chrome

### ISSUE THREAD

**Issue by Studio384** [twbs/bootstrap#18763]
In Microsoft Edge 13, the date field in Bootstrap 4 is to large compared to the other fields.

http://jsbin.com/lakoterure/1/edit?html,output

**Comment by twbs-lmvtfy** [comment]
Hi @Studio384!

You appear to have posted a live example (http://jsbin.com/gopaqecowe/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 14, column 1: [E007](https://github.com/twbs/bootlint/wiki/E007): Only one copy of Bootstrap's JS should be included; currently the webpage includes both bootstrap.js and bootstrap.min.js
* line 26, column 5: [E007](https://github.com/twbs/bootlint/wiki/E007): Only one copy of Bootstrap's JS should be included; currently the webpage includes both bootstrap.js and bootstrap.min.js

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by Studio384** [comment]
http://jsbin.com/lakoterure/1/edit?html,output

**Comment by patrickhlauke** [comment]
it appears to be the same size as in Chrome.
![capture](https://cloud.githubusercontent.com/assets/895831/12112629/ae398e86-b395-11e5-83b5-619ce087d1e8.PNG)

**Comment by cvrebert** [comment]
Possible duplicate of #17308.
Probably we should add an example with all of the textual input types to http://v4-alpha.getbootstrap.com/components/forms/ for testing purposes.

**Comment by Studio384** [comment]
> Possible duplicate of #17308.

Not a duplicate I think, that's about the small variants having a different size. The date fields get a different lineheight:
```css
@media screen and (-webkit-min-device-pixel-ratio:0) {
    input[type=date].form-control, input[type=time].form-control, input[type=datetime-local].form-control, input[type=month].form-control {
        line-height: 2.25rem;
    }
}
```

While other fields have line-height ```1.5```.
